politeiawww/www.go

```diff
@@ -639,8 +637,9 @@ done:
 	// Close user db connection
 	p.db.Close()
 
-	if p.pubSubDcrdata != nil && p.pubSubDcrdata.client != nil {
-		p.pubSubDcrdata.client.Stop()
+	// Shutdown all dcrdata websockets
+	if p.wsDcrdata != nil {
```

Comment:
Do we not need to verify that p.wsDcrdata.client is not nil here?

Reply:
No because `wsDcrdata` is initialized using the function `newWSDcrdata`, which ensures that the client is setup correctly.  If it's being initialized without using `newWSDcrdata` then its being initialized incorrectly.

Reply:
Cool thanks for clarifying.  Looks good.